ganga/GangaLHCb/Lib/RTHandlers/GaudiExecRTHandlers.py

```diff
@@ -135,13 +132,15 @@ def collectPreparedFiles(app):
         if isinstance(file_, LocalFile):
             if file_.namePattern == 'data.py':
                 raise ApplicationConfigurationError(
-                    "You should not name any inputfiles 'data.py' to avoid conflict with the generated inputdata. Please rename the file and submit again.")
+                    "You should not name any inputfiles 'data.py' to avoid conflict with the generated inputdata."
+                    " Please rename the file and submit again.")
             shutil.copy(os.path.join(file_.localDir, os.path.basename(file_.namePattern)), shared_dir)
             input_files.append(os.path.join(shared_dir, file_.namePattern))
         elif isinstance(file_, str):
             if 'data.py' in file_:
                 raise ApplicationConfigurationError(
-                    "You should not name any inputfiles 'data.py' to avoid conflict with the generated inputdata. Please rename the file and submit again.")
+                    ("You should not name any inputfiles 'data.py' to avoid conflict with the generated inputdata."
```

Comment:
Hard to make out from the diff here but is there an extra open bracket? i.e. one on this line and one on the unchanged line above

Reply:
good spot